roles/win_nodes/kubernetes_patch/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Ensure that user manifests directory exists
```

Comment:
I'm not sure to understand why we need to apply patches from here

Reply:
It is only required for cases where we don't have a .yml to modify before creating the kube-proxy, I think kubeadm creates the daemonset, so I had to patch after creation.

Reply:
ok thanks